web/frontend/package.json

```diff
@@ -17,7 +17,7 @@
     "@improbable-eng/grpc-web": ">=0.15",
     "@viamrobotics/prime": ">=0.5",
     "@viamrobotics/rpc": ">=0.1",
-    "@viamrobotics/sdk": "0.3.1-next.1",
+    "@viamrobotics/sdk": "^0.3.2-rc.0",
```

Comment:
do we feel safe enough with the SDK to not lock the version at this point (`^`)? Will leave it to your discretion